lib/rules/declaration-block-trailing-semicolon/__tests__/index.js

```diff
@@ -62,6 +62,58 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: ['always', { ignore: 'single-declaration' }],
+	fix: true,
+
+	accept: [
+		{
+			code: 'foo { property: value }',
+			description: 'single declaration without trailing semicolon',
+		},
+		{
+			code: 'foo { property: value; }',
+			description: 'single declaration with trailing semicolon',
+		},
+		{
+			code: '@keyframes name { from { property: 0 } to { property: 1; } }',
```

Comment:
```suggestion
			code: '@keyframes foo { from { top: 0px } to { top: 0px; } }',
```

Reply:
`code: '@keyframes foo { from { top: 0px } to { top: 1px; } }',`